config.py

```diff
@@ -80,56 +81,10 @@
-                  'magic wand', 'black king bar', 'observer and sentry wards', 'butterfly', 'clarity', 'shadow amulet',
-                  'skull basher', 'boots of speed', 'helm of iron will', 'medallion of courage', 'bloodstone',
-                  'divine rapier', 'gauntlets of strength', 'dagon', 'ghost scepter', 'boots of travel', 'moon shard',
-                  'abyssal blade', 'vitality booster', 'ring of protection', 'blade of alacrity', 'ring of health',
-                  'cloak', 'shadow blade', 'diffusal blade', 'tango', 'dust of appearance', 'belt of strength',
-                  'hurricane pike', 'vladmir s offering', 'tranquil boots', 'javelin', 'meteor hammer', 'broadsword',
-                  'ultimate orb'}
-
-# Note: Get them from here
-# https://dota2.gamepedia.com/api.php?action=cargoquery&tables=heroes&fields=title&where=game+IS+NULL&limit=500&format=json
-HERO_NAME_RESPONSES = {'silencer', 'phantom assassin', 'clinkz', 'huskar', 'juggernaut', 'crystal maiden', 'pudge',
-                       'disruptor', 'queen of pain', 'wraith king', 'spectre', 'templar assassin', 'warlock',
-                       'earth spirit', 'viper', 'slark', 'weaver', 'alchemist', 'treant protector', 'axe', 'tidehunter',
-                       'invoker', 'kunkka', 'keeper of the light', 'undying', 'phoenix', 'terrorblade', 'doom',
-                       'broodmother', 'death prophet', 'earthshaker', 'mirana', 'storm spirit', 'bounty hunter',
-                       'clockwerk', 'lina', 'magnus', 'lifestealer', 'enigma', 'windranger', 'dark seer', 'drow ranger',
-                       'tiny', 'chaos knight', 'vengeful spirit', 'nyx assassin', 'ancient apparition', 'tusk',
-                       'ember spirit', 'io', 'outworld devourer', 'lion', 'underlord', 'lone druid', 'ursa', 'batrider',
-                       'riki', 'sven', 'ogre magi', 'beastmaster', 'anti mage', 'morphling', 'medusa', 'arc warden',
-                       'shadow demon', 'naga siren', 'slardar', 'bloodseeker', 'winter wyvern', 'leshrac', 'lycan',
-                       'omniknight', 'witch doctor', 'shadow shaman', 'sand king', 'necrophos', 'faceless void',
-                       'pangolier', 'grimstroke', 'dazzle', 'visage', 'spirit breaker', 'centaur warrunner',
-                       'monkey king', 'jakiro', 'dragon knight', 'abaddon', 'pugna', 'dark willow', 'night stalker',
-                       'luna', "nature s prophet", 'lich', 'bane', 'mars', 'phantom lancer', 'troll warlord', 'chen',
-                       'techies', 'skywrath mage', 'enchantress', 'razor', 'gyrocopter', 'tinker', 'zeus', 'meepo',
-                       'rubick', 'elder titan', 'brewmaster', 'venomancer', 'shadow fiend', 'puck', 'legion commander',
-                       'sniper', 'oracle', 'timbersaw', 'bristleback', 'snapfire', 'void spirit'}
+# Hero and item responses not hardcoded here
+# Drawbacks: Can't be tweaked
+HERO_NAME_RESPONSES = request_hero_name_set()
```

Comment:
`request_cargo_set` can be one method with the url as parameter, as rest of the logic is the same.

Reply:
Good catch, fixed in https://github.com/Jonarzz/DotaResponsesRedditBot/pull/41/commits/c370f726561bfe9b60a2ec63a0d868aa89905b8a.